.github/workflows/lint.yaml

```diff
@@ -33,3 +33,22 @@ jobs:
       - name: Fail if lint failed
         if: steps.linter.outputs.checks-failed > 0
         run: exit 1
+  py_lint:
+    name: Python format and lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install black
+      - name: Run Black
+        run: |
+          black . --check --exclude tools/splat/ --exclude tools/splat_ext/ --exclude diff.py
```

Comment:
Is there a reason splat_ext is excluded? Those are our custom segments, though getting path stuff set up correctly might be tricky for that directory

Reply:
In https://github.com/pmret/papermario/issues/949 [nanaian](https://github.com/nanaian) calls out that "splat/diff.py/friends" should be excluded

Reply:
Yeah, splat ext is distinct from splat though

Reply:
Removed the exclude on splat_ext